packages/shared/src/components/footer/FooterLinks.tsx

```diff
@@ -9,33 +9,35 @@ export type FooterLinksProps = {
 
 export const FooterLinks = ({ className }: FooterLinksProps): ReactElement => {
   return (
-    <ul
-      className={classNames(
-        className,
-        'flex flex-row flex-wrap justify-center gap-3 text-text-tertiary typo-caption1',
-      )}
-    >
-      <li>&copy; {new Date().getFullYear()} Daily Dev Ltd.</li>
-      <li>
-        <a href={contentGuidelines} target="_blank" rel={anchorDefaultRel}>
-          Guidelines
-        </a>
-      </li>
-      <li>
-        <a href={`${webappUrl}posts`}>Explore</a>
-      </li>
-      <li>
-        <a href={`${webappUrl}tags`}>Tags</a>
-      </li>
-      <li>
-        <a href={`${webappUrl}sources`}>Sources</a>
-      </li>
-      <li>
-        <a href={`${webappUrl}squads`}>Squads</a>
-      </li>
-      <li>
-        <a href={`${webappUrl}users`}>Leaderboard</a>
-      </li>
-    </ul>
+    <footer>
```

Comment:
Those links are always in the end of the page, footer is a better tag element imo